tests/test_model.py

```diff
@@ -703,35 +710,39 @@ def test_generate(attention_impl, device, alibi):
     )
     mpt = MPTForCausalLM(hf_config)
     mpt.eval()
-    mpt = device.module_to_device(mpt)
+    mpt = composer_device.module_to_device(mpt)
 
     # padding on the left of the input
     left_padding_input_ids = torch.tensor(
         [[50256, 50256, 50256, 11274, 16390, 11],
          [50256, 50256, 50256, 11274, 16390, 11]])
-    left_padding_input_ids = device.tensor_to_device(left_padding_input_ids)
+    left_padding_input_ids = composer_device.tensor_to_device(
+        left_padding_input_ids)
     left_padding_attention_mask = torch.tensor([[0, 0, 0, 1, 1, 1],
                                                 [0, 0, 0, 1, 1, 1]])
-    left_padding_attention_mask = device.tensor_to_device(
+    left_padding_attention_mask = composer_device.tensor_to_device(
         left_padding_attention_mask)
 
     # no padding in the input
     no_padding_input_ids = torch.tensor([[11274, 16390, 11], [11274, 16390,
                                                               11]])
-    no_padding_input_ids = device.tensor_to_device(no_padding_input_ids)
+    no_padding_input_ids = composer_device.tensor_to_device(
+        no_padding_input_ids)
     no_padding_attention_mask = torch.tensor([[1, 1, 1], [1, 1, 1]])
-    no_padding_attention_mask = device.tensor_to_device(
+    no_padding_attention_mask = composer_device.tensor_to_device(
         no_padding_attention_mask)
 
     # a single batch with different amounts of left padding in the input
     batched_input_ids = torch.tensor([[50256, 50256, 50256, 11274, 16390, 11],
                                       [50256, 50256, 16, 11274, 16390, 11]])
-    batched_input_ids = device.tensor_to_device(batched_input_ids)
+    batched_input_ids = composer_device.tensor_to_device(batched_input_ids)
     batched_attention_mask = torch.tensor([[0, 0, 0, 1, 1, 1],
                                            [0, 0, 1, 1, 1, 1]]).bool()
-    batched_attention_mask = device.tensor_to_device(batched_attention_mask)
+    batched_attention_mask = composer_device.tensor_to_device(
+        batched_attention_mask)
 
-    with get_precision_context('amp_bf16' if device.name == 'gpu' else 'fp32'):
+    with get_precision_context('amp_bf16' if composer_device.name ==
+                               'gpu' else 'fp32'):
```

Comment:
since this isn't marked as a gpu test, `if device.name == 'gpu'` will always be False?

Reply:
that seems right. isn't harming anything though so im just going to leave it